### DISCUSSION TITLE
Enforce blank line after code block if already contains any white line

### DISCUSSION THREAD

**Discussion by SirVizzy** [eslint/eslint#18637] [Q&A] [answered]
I am looking to set up my ESLint in a way that keeps the code scannable. Whenever any code block has a new line in it, a new line must be placed at the top of the codeblock. I've been looking at `stylistic/padding-line-between-statements` but can't seem to figure it out.

Basically what I am looking for is the following.

**Incorrect**
```ts
const incorrectFunction = () => {
  const process = () => {
    const a = 1;
    const b = 2;
  
    return a + b;
  }


  return process();
}
```

**Correct**
```ts
const correctFunction = () => {

  const process = () => {

    const a = 1;
    const b = 2;
  
    return a + b;
  }


  return process();
}
```

**Comment by fasttime** [comment, marked as answer]
It looks like the rule you are using comes from the [ESLint Stylistic](https://github.com/eslint-stylistic) plugin. We don't maintain that plugin, so you could probably get better answers if you ask for support there.

**Reply by SirVizzy** [reply]
Ah. Alright. For anyone lingering around this discussion in the future; moved [here](https://github.com/eslint-stylistic/eslint-stylistic/discussions/456).
